libretroshare/src/rsserver/p3face-server.cc

```diff
@@ -108,21 +107,16 @@ RsServer::RsServer() :
 	msgSrv = NULL;
 	chatSrv = NULL;
 	mStatusSrv = NULL;
-    mGxsTunnels = NULL;
+	mGxsTunnels = NULL;
 
-    mMin = 0;
-    mLoop = 0;
+	mLastts = getCurrentTS();
+	mTickInterval = maxTickInterval ;
+	mAvgRunDuration = 0;
+	mLastRunDuration = 0;
```

Comment:
At first glance I'd say it is better to initialize those in initializers list like `coreMutex` and `coreReady`